src/types.ts

```diff
@@ -32,8 +34,22 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface NetworksToAddresses {
+  [address: string]: Maybe<AddressesToTokenApiInfo>;
+}
+
+export interface AddressesToTokenApiInfo {
+  [address: string]: Maybe<TokenApiInfo>
+}
+
+export interface TokenApiInfo {
```

Comment:
Could we move `NetworksToAddresses` `AddressesToTokenApiInfo` and `TokenApiInfo` to the `src/actions/getPaymentTokenInfo.ts` file? These seem very specific to a single operation that we do, and are only ever used internally. We should avoid putting internal/private types into our `types.ts` file (or module) since the user will be importing all of these and we don't want to confuse them with internal/implementation details which they have no control over.